sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -1,8 +1,12 @@
 # Azure Form Recognizer client library for Java
-Form Recognizer is a cloud-based cognitive service that uses machine learning technology to identify and extract text, 
-key-value pairs and table data from form documents.
+Azure Cognitive Services Form Recognizer is a cloud service that uses machine learning to recognize text and table data
+from form documents. Form Recognizer is made up of the following services:
 
-[Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_reference_doc] | [Product Documentation][product_documentation] | [Samples][samples_readme]
+* Custom models - Extract name/value pairs and table data from forms. These models are trained with your own data, so they're tailored to your forms.
+* Prebuilt receipt model - Extract data from USA sales receipts using a prebuilt model.
+* Layout API - Extract text and table structures, along with their bounding box coordinates, from documents.
```

Comment:
For now, I think we're using Content API for this, then saying in the explanation that it corresponds to the service's Layout endpoint